config/blah.config.template

```diff
@@ -80,7 +80,7 @@ blah_require_proxy_on_submit=
 
 #Path where PBS executables are located 
 # NOTE: this path is computed many times; I worry about the overhead here.  -BB, 12-13-2012
-pbs_binpath=`which qsub 2>/dev/null|sed 's|/[^/]*$||'`
+pbs_binpath=/usr/bin
```

Comment:
This would only affect those sites that haven't edited their `blah.config` since it's marked config noreplace in the spec but I think these cases will be few and far between because of osg-configure modifying blah.config (@matyasselmeci does osg-configure always modify `blah.config`?).

We can send an announcement and bump the minor version number, 1.19.0!